Article/Article.js

```diff
@@ -1,29 +1,96 @@
+/* eslint-disable max-len */
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class.
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = '-----';
+
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    // this.domElement.classList.toggle('article-open');
+    if (this.domElement.style.height === '400px') {
+      TweenMax.to(this.domElement, 0.5, { height: 50, ease: Power3.easeOut });
```

Comment:
I think its probably not a good idea to check for height because it's not overly reliable, a good way to do it might be something like this:

```constructor(domElement) {
    this.closed = true;
    // other refs
}```

then:

```expandArticle() {
    this.closed = !this.closed;
    if (this.closed) {
       //logic here
    }
```